### PR TITLE
Notebook detailing HGDP sample selection.

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
View here with nbviewer: https://nbviewer.jupyter.org/github/populationgenomics/ancestry/blob/HGDP_SampleSelection/scripts/HGDP_SampleSelection.ipynb

### CODE REVIEW

File: scripts/HGDP_SampleSelection.ipynb [hunk 1]
```diff
@@ -0,0 +1,1716 @@
+{
```
Comment (pdiakumis) [inline scripts/HGDP_SampleSelection.ipynb:1]: <p>Another way to do this is with f-strings (https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python):</p><p> </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; x = 123
&gt;&gt;&gt; print(f'After filtering, {x} samples remain')
After filtering, 123 samples remain
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/populationgenomics/ancestry/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (pdiakumis) [inline scripts/HGDP_SampleSelection.ipynb:1]: <p>Nice!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/populationgenomics/ancestry/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (pdiakumis) [inline scripts/HGDP_SampleSelection.ipynb:1]: <p>By the way, I don't think any Papuan samples survive this filtration, which is unfortunate.. If we go ahead with downloading Papuan CRAMs, they'll end up getting filtered out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/populationgenomics/ancestry/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

**Review by pdiakumis — APPROVED** [review]
Looks great @KatalinaBobowik!
I commented via https://app.reviewnb.com/populationgenomics/ancestry/pull/3/discussion/, so will be good to see if that helps - I don't think there's another way to comment on large files in GitHub itself.

**Comment by KatalinaBobowik** [comment]
Fantastic, thanks for the comments Peter. Good to know you can do it through reviewnb

**Pull request merged**